### PR TITLE
Migrations

### LINKED ISSUE #7: Create User Table + Model (with tests)
- Refer to schema design on [miro](https://miro.com/app/board/o9J_lbZPdcE=/)

### LINKED ISSUE #8: Create Events Table + Model (with tests)
- Refer to schema design on [miro](https://miro.com/app/board/o9J_lbZPdcE=/)

### LINKED ISSUE #10: Create Questions Table + Model (with tests)
- Refer to schema design on [miro](https://miro.com/app/board/o9J_lbZPdcE=/)

### LINKED ISSUE #20: Create Eras Table + Model (with tests)
- Refer to schema design on [miro](https://miro.com/app/board/o9J_lbZPdcE=/)

### PR DESCRIPTION
**Pull request by AngelaGuardia**
### Code Highlights:
 * Created migrations for Users, Eras, Events tables
 * Added model validations, relationships and relevant tests for Users, Eras, and Events
 
### Where should the reviewer start?
 * Review the model tests in the spec folder
 * Review `/db` folder
 * 
### Have Tests Been Added?
- [ ] No.
- [ ] Yes, but all tests are not passing.
- [x] Yes, and all are passing.
### Any background context you want to provide?
- Group paired
### Issues:
* Closes #10
* closes #7
* closes #8 
* closes #20
### Tracking Consistency:
- [ ] added appropriate labels
- [ ] My code follows the code style of this project and has removed all unnecessary annotations
- [ ] I have added comments on my pull request, particularly in hard-to-understand areas
- [ ] looked at PR preview to check spelling, syntax, formatting, and completion
- [x ] Rubocop Violations:

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require: rubocop-rails
```
Comment (dhalverson) [inline .rubocop.yml:1]: We can fine tune these as we keep working here.

**Review by LHJE — APPROVED** [review]
All of this looks great!

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
```
Comment (dhalverson) [inline app/models/user.rb:1]: Looks good

**Review by dhalverson — APPROVED** [review]
We worked on these together! All good

**Pull request merged**